BUILD.bazel

```diff
@@ -25,6 +25,7 @@ lines_sorted_test(
 
 gazelle(
     name = "gazelle",
+    prefix = "github.com/bazelbuild/rules_go",
```

Comment:
I thought we were doing this with a directive rather than an attribute now?

Reply:
Done. I'm a bit ambivalent about this though. Should we remove all attributes on the new `gazelle` rule (assuming they can be replaced by directives)?

Reply:
If gazelle is doing the rewriting, yes, if not, lets add debug prints that strongly encourage people to switch to the directives.

Reply:
Gazelle will at least rewrite the rule. It could do the attributes at the same time. bazelbuild/bazel-gazelle#27